app/src/main/java/com/myvetpath/myvetpath/ViewDraftsActivity.java

```diff
@@ -123,6 +124,7 @@ protected void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
```

Comment:
In this activity, we should have it requery the data (maybe on the onResume) and update the recyclerview. The reason being is that if you edit the title of the submission and then go back to this activity, the title won't be updated. Same goes for the time. Lastly, we can probably get rid of the "pending" part since it's a draft.